packages/hooks/useNSNameInfo.ts

```diff
@@ -10,18 +10,23 @@ import {
 import { GnoNetworkInfo, NetworkKind, getNetwork } from "@/networks";
 import { getCosmosNameServiceQueryClient } from "@/utils/contracts";
 import { extractGnoJSONString, extractGnoString } from "@/utils/gno";
+import { ProfileData } from "@/utils/upp";
+
+export const GNO_CONTRACT_FIELD = {
+  // Standard fields
+  DISPLAY_NAME: "DisplayName",
+  BIO: "Bio",
+  AVATAR: "Avatar",
+
+  // Extra fields
+  BANNER: "Ext_Banner",
```

Comment:
Not sure we need to prefix, this could actually become standard later and if we prefix now, we will have to support two fields